docs/cache-encryption-keys.md

```diff
@@ -0,0 +1,69 @@
+---
+id: cache-encryption-keys
+title: Customer Managed Encryption Keys
+sidebar_label: Customer Managed Encryption Keys
+---
+
+BuildBuddy allows you to provide a managed key to be used to encrypt cache artifacts at rest.
+
+BuildBuddy currently support encryption keys stored in Google Cloud Platform (GCP) KMS or Amazon Web Services (AWS) KMS.
```

Comment:
s/support/supports

also, more because the grammar nut in me is mad about it, i just wanted to point out that `customer-managed` is probably the more grammatically correct way to write this, so naturally some providers use `customer managed` (AWS) and others user `customer-managed` (GCP).

Reply:
updated on both counts